components/coachindicator/stories/coachindicator.mdx

```diff
@@ -0,0 +1,24 @@
+import { Canvas, ArgTypes, Meta, Description, Title } from '@storybook/blocks';
+
+import * as CoachIndicator from './coachindicator.stories';
+
+<Meta of={CoachIndicator} title="Docs" />
+
+<Title of={CoachIndicator} />
+<Description of={CoachIndicator} />
+
+### Migration Guide
+
+Coach Indicator is now a separate component from [Coach Mark.](/docs/components-coach-mark--docs)
```

Comment:
```suggestion
Coach Indicator is primarily used along with the [Coach Mark](/docs/components-coach-mark--docs) component.
```

Reply:
Hehehe suggestion inception! Reminder to use sentence case even for component names

```suggestion
### Migration Guide

Coach indicator is now a separate component from [Coach mark.](/docs/components-coach-mark--docs)
```

Reply:
great catches from both! https://github.com/adobe/spectrum-css/pull/2793/commits/6b0faaeb26da1a3fb56c138f05f1dcbf436a59ff